src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/OffenderAliasEntity.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.probation.courtcaseservice.jpa.entity;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffenderAliasEntity {
+
+    private final LocalDate dateOfBirth;
+    private final String firstName;
+    private final String surname;
+    private final String gender;
```

Comment:
Do we have full list of values for this as the values are different in the DefendantEntity where the gender is referred to as sex and the values are "MALE", "FEMALE", "NOT_KNOWN", "NOT_SPECIFIED" should we create a new enum for gender to accept Male, Female.. ? @jhevans